### PR TITLE
fix: remove projectId from create feature form

### PR DESCRIPTION
**Pull request by ykhedher**

### CODE REVIEW

**Comment by ykhedher** [comment]
[bug: we should not send projectId in body when creating a feature toggle](https://trello.com/c/mn9Uhu9i/832-bug-we-should-not-send-projectid-in-body-when-creating-a-feature-toggle)

**Review by FredrikOseberg — APPROVED** [review]
LGTM

File: src/component/feature/CreateFeature/hooks/useFeatureForm.ts [hunk 1]
```diff
@@ -42,7 +42,6 @@ const useFeatureForm = (
         return {
             type: type,
             name: name,
-            projectId: project,
```
Comment (FredrikOseberg) [inline src/component/feature/CreateFeature/hooks/useFeatureForm.ts:45]: What do we actually use project for now?

**Review by FredrikOseberg — CHANGES_REQUESTED** [review]
On closer inspection we need to validate that we are using the projectId from the dropdown to send the request to the correct URL.

File: src/component/feature/CreateFeature/hooks/useFeatureForm.ts (hunk 1 above)
Comment (ivarconr) [reply src/component/feature/CreateFeature/hooks/useFeatureForm.ts:45]: The form will probably still need to capture the projectId in order to call the correct API endpoint to create the feature toggle (projectId is part of the URL). We should just no send is as part of the json body anymore.

Comment (ykhedher) [reply src/component/feature/CreateFeature/hooks/useFeatureForm.ts:45]: We just removed the projectId from the body payload but we still keep it in the URL.

**Review by FredrikOseberg — APPROVED** [review]
LGTM

**Pull request merged**